### ISSUE TITLE
Valid grasps not displayed in scene generation

### ISSUE THREAD

**Issue by schnellerblitz99** [NVlabs/acronym#4]
Hey, 
I have an issue with the scene generation. Everything seems to work fine, except the valid grasps are not displayed when generating a scene(--show_grasps is set). I am working with Ubuntu 20.04 and conda environment. Do you have an idea why it is not working? Thanks!

**Comment by clemense** [comment]
Hi, this was tested under 18.04. Is there any output/error message that you get when executing the code?

Another possibility that comes to mind is that there might be no valid grasps in the particular scene you are looking at (if all of them are in collision). You can test this by e.g. adding `print(len(gripper_markers))` in line https://github.com/NVlabs/acronym/blob/main/scripts/acronym_generate_scene.py#L115 and check that the output is greater than zero.

**Comment by schnellerblitz99** [comment]
Hello, 
thank you for you fast reply. So it works under Ubuntu20.04, too. I just had to uncomment or exit the first window which appears containing only the objects in the scene. 
![Screenshot from 2021-04-15 09-52-21](https://user-images.githubusercontent.com/48187015/114834224-a3b15b80-9dd0-11eb-8bf2-00973d9279fc.png)
So I guess it is normal, that not every possible, collision-free grasp is displayed?
Again, thanks!

**Comment by clemense** [comment]
Yes, not every grasps is shown, the visualization will become very slow, but you can increase the parameter `--num_grasps_per_object`, see here:
https://github.com/NVlabs/acronym/blob/cd20a7cc5b5b1f3f5d9488be6ff8698651d96590/scripts/acronym_generate_scene.py#L61
